Gemfile

```diff
@@ -155,6 +155,7 @@ gem 'therubyracer', '~> 0.12.0'
 
 # Feature flags
 gem 'alaveteli_features', :path => 'gems/alaveteli_features'
+gem 'flipper-ui'
```

Comment:
Have a read of the Gemfile documentation above. Will want to add a pessimistic constraint.